pages/report.tsx

```diff
@@ -65,7 +67,7 @@ enum FormStep {
   AuthorInfo,
 }
 
-function useInitialFormValues(): UnsubmittedFormResponse {
+function useInitialFormValues(storageFolder: string): UnsubmittedFormResponse {
```

Comment:
In Typescript, you can use a type's key to specify the specific type of one of an object-type's properties:

```suggestion
function useInitialFormValues(storageFolder: UnsubmittedFormResponse["storage_folder"]): UnsubmittedFormResponse {
```